packages/create/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@sku/create",
+  "version": "0.0.0",
+  "description": "Create sku projects",
+  "private": true,
+  "bin": {
+    "create-sku": "./dist/cli.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=18"
+  },
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json",
+    "test": "vitest"
```

Comment:
Tests will be handles by the top-level `test` script so this shouldn't be necessary.